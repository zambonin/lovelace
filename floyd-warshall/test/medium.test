#!/bin/sh

./floyd-warshall/fw /dev/stdin <<EOF
3 5
0 1 4
2 1 5
2 1 6
1 2 234
0 2 400
EOF
