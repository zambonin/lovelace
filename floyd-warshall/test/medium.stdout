All pair shortest distances
      0      4    238
    inf      0    234
    inf      5      0
